highlight/RON.sublime-syntax

```diff
@@ -0,0 +1,148 @@
+        - meta_scope: meta.structure.array.ron
+        - match: '\]'
+          scope: punctuation.section.array.end.ron
+          pop: true
+        - include: value
+        - match: ','
+          scope: punctuation.separator.array.ron
+        - match: '[^\s\]]'
+          scope: invalid.illegal.expected-array-separator.ron
+  comments:
+    - match: /\*\*(?!/)
+      scope: punctuation.definition.comment.ron
+      push:
+        - meta_scope: comment.block.documentation.ron
+        - match: \*/
+          pop: true
+    - match: /\*
+      scope: punctuation.definition.comment.ron
+      push:
+        - meta_scope: comment.block.ron
+        - match: \*/
+          pop: true
+    - match: (//).*$\n?
+      scope: comment.line.double-slash.js
+      captures:
+        1: punctuation.definition.comment.ron
+  constant:
+    - match: \b(?:true|false)\b
+      scope: constant.language.ron
+  number:
+    # handles integer and decimal numbers
+    - match: |-
+        (?x:         # turn on extended mode
+          -?         # an optional minus
+          (?:
+            0        # a zero
+            |        # ...or...
+            [1-9]    # a 1-9 character
+            \d*      # followed by zero or more digits
+          )
+          (?:
+            (?:
+              \.     # a period
+              \d+    # followed by one or more digits
+            )?
+            (?:
+              [eE]   # an e character
+              [+-]?  # followed by an option +/-
+              \d+    # followed by one or more digits
+            )?       # make exponent optional
+          )?         # make decimal portion optional
+        )
+      scope: constant.numeric.ron
+  object:
+    - match: '[A-Z][a-zA-Z]*'
+      scope: entity.name.class.ron
+    - match: '\('
+      scope: punctuation.section.dictionary.begin.ron
+      push:
+        - meta_scope: meta.structure.entity.ron
+        - match: '\)'
+          scope: punctuation.section.dictionary.end.ron
+          pop: true
+        - match: '[a-z_][a-zA-Z0-9_]*'
+          scope: entity.name.tag.ron
+        - match: '\:'
+          scope: punctuation.separator.dictionary.key-value.ron
+        - include: value
+        #- match: '[^\s,]'
+        #  scope: invalid.illegal.expected-dictionary-separator.ron
+        #- match: '[^\s\)]'
+        #  scope: invalid.illegal.expected-dictionary-separator.ron
+  dictionary:
+    - match: '\{'
+      scope: punctuation.section.dictionary.begin.ron
+      push:
+        - meta_scope: meta.structure.dictionary.ron
+        - match: '\}'
+          scope: punctuation.section.dictionary.end.ron
+          pop: true
+        # only string keys are supported by this highlighter atm
+        - match: '"'
+          scope: punctuation.definition.string.begin.ron
+          push:
+            - meta_scope: meta.structure.dictionary.key.ron string.quoted.double.ron
+            - include: inside-string
+        - include: comments
+        - match: ':'
```

Comment:
Later this this context should be pushed as soon as a value (the key) has been parsed, right?